roles/ceph-osd/defaults/main.yml

```diff
@@ -4,15 +4,15 @@ ceph_osd:
   pkgs:
     - parted
 
-  # ONLY SET THIS TO TRUE IF YOU KNOW WHAT YOU'RE DOING.
-  # This will likely only be used on a fresh install.
-  destroy_devices_no_matter_what: false
+  # choose one scenario
+  journal_collocation: false

```

Comment:
I was going to amend this and set `journal_collocation` to true. I have a comment but I also need to add a check to fail if they're both false. Also, I'd like to leave the `raw_multi_journal` code in here. We may not use it right away but something tells me we will down the line. And it's been tested.